source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
There is no way to check at compile time that the values in `values` are `Float`, and any other type of values will cause the program to crash at run time. As nice as it would be to use varargs, don't - use a `FloatMatrix` or something instead.